components/org.wso2.carbon.database.utils/src/main/java/org/wso2/carbon/database/utils/jdbc/NamedPreparedStatement.java

```diff
@@ -60,6 +60,46 @@
     public NamedPreparedStatement(Connection connection, String sqlQuery, Map<String, Integer> repetition)
             throws SQLException {
 
+        this(connection, sqlQuery, repetition, "ID");
+    }
+
+    /**
+     * Create a named prepared statement.
+     *
+     * @param connection Database connection to be used.
+     * @param sqlQuery   Underlying SQL query.
+     * @throws SQLException SQL Exception.
+     */
+    public NamedPreparedStatement(Connection connection, String sqlQuery) throws SQLException {
+
+        this(connection, sqlQuery, new HashMap<String, Integer>());
+    }
+
+    /**
+     * Create a named prepared statement with a specified column value to be returned.
+     *
+     * @param connection Database connection to be used.
+     * @param sqlQuery   Underlying SQL query.
+     * @param columnName Name of the unique column to be returned upon query success.
+     * @throws SQLException SQL Exception.
+     */
+    public NamedPreparedStatement(Connection connection, String sqlQuery, String columnName) throws SQLException {
```

Comment:
let's have this as columnNameToReturn or something similar

Reply:
Changed to `returningColumnName` in a221aafb2abf74073c410c616a91810d2b9f7ffc